main/views.py

```diff
@@ -13,7 +15,8 @@ def index(request):
 
 @login_required()
 def homepage(request):
-    return render(request, 'main/homepage.html')
+    if DogOwner.objects.filter(user=request.user).exists():
```

Comment:
And what happens if it is not a dog_owner?
anyway, this needs a test.

Reply:
My question was aimed toward the fact that *right now* the homepage leads somewhere only if the user is a DogOwner.
if there is only a dogowner, no real need for this statement.
if there is not only a dogowner, this needs an "elif". 
OR... maybe it should lead to a dedicated landing\about page.

Reply:
I've added an elif statement for the daycare, redirected (for now) to the "Coming soon" homepage,
and if the logged in user is not a dog owner and not a daycare, they'll be redirected to the "about" page.

Reply:
There are 2 flaws here: 
1. If this stays as a `login_required`, there is no need for the last return. do you want a not-logged-in user to not be able to view the homepage?
2. missing a test.

Reply:
1) Fixed . 
2) Test for both cases were added.